src/BPEssentials/Commands/Server/Save.cs

```diff
@@ -12,27 +11,20 @@ public class Save : Command
     {
         public void StartSaveTimer()
         {
-           
-                var Tmer = new System.Timers.Timer(); // TODO: Disposing the timer seems to break
-                Tmer.Elapsed += (sender, e) => Run();
-                Tmer.Interval = 15 * 60 * 1000;
-                Tmer.Enabled = true;
-            
+            int interval = 15 * 60 * 1000;
```

Comment:
Lets add this to the Config while we are at it

Reply:
Sure